extensions/common/http/jersey-core/src/testFixtures/java/org/eclipse/edc/web/jersey/testfixtures/RestControllerTestBase.java

```diff
@@ -35,14 +35,12 @@
  */
 public abstract class RestControllerTestBase {
 
-    private JettyService jetty;
     protected final int port = getFreePort();
     protected Monitor monitor = mock(Monitor.class);
-
-    protected abstract Object controller();
+    private JettyService jetty;
 
     @BeforeEach
-    void setup() {
+    final void setup() {
```

Comment:
the test setup fails if the subclass also declares a `setup` method, which is quite common for `@BeforeEach`

Reply:
we could also call it like `startJetty` so there won't be habit changes